### DISCUSSION TITLE
Negative prompt?

### DISCUSSION THREAD

**Discussion by magickaito** [oobabooga/text-generation-webui#3045] [Q&A]
Is there anything equivalent to the "negative prompt" in Stable Diffusion Automatic1111's UI? 

Like if a user clearly downvoted a few chat message samples, how can I supply this (ideally in api) as negative samples?

**Comment by viperwasp** [comment]
There seems to be a negative prompt field now? Never saw it before. Anyone use it? Any tips or limitations? Is it for all models? 

It seems to disappear for llama.ccp and llama.cpp-hf  so it can't be used for GGML models?
